cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +91,26 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
+  OUTPUT_NAME "libigfxcmrt${TARGET_MODIFIER}"
   )
+
+# set the version, aligned with media driver
+if(NOT DEFINED MEDIA_VERSION OR MEDIA_VERSION STREQUAL "")
```

Comment:
This is already done here:
https://github.com/intel/media-driver/blob/f9b1c19c36ead3cca95295581f0fee8b8b99a137/CMakeLists.txt#L33
Remove.

Reply:
This CMakeFile can be used directly sometime, like by the build_linux.sh. This is what we need to generate the CM SDK package.

Reply:
What is build_linux.sh? I cannot find it in the current repository.  Please avoid wrapping build targets in scripts.

Reply:
build_linux.sh is in cmrtlib folder. But you are right, we can remove that script.
Back to this topic, With this checking, we can also make the cmrtlib alone without media_driver.